articles/sentinel/automate-responses-with-playbooks.md

```diff
@@ -31,7 +31,7 @@ Many, if not most, of these alerts and incidents conform to recurring patterns t
 
 A playbook is a collection of these remediation actions that can be run from Azure Sentinel as a routine. A playbook can help [**automate and orchestrate your threat response**](tutorial-respond-threats-playbook.md); it can be run manually or set to run automatically in response to specific alerts or incidents, when triggered by an analytics rule or an automation rule, respectively.
 
-Playbooks are created and applied at the subscription level, but the **Playbooks** tab (in the new **Automation** blade) displays all the playbooks available across any selected subscriptions.
+Playbooks are created inside a resource group, but the **Playbooks** tab (in the new **Automation** blade) displays all the playbooks available across any selected subscriptions.
```

Comment:
This statement and the one it would replace are both true. Yes, playbooks are created inside a resource group. However, they are visible and applicable within a given subscription. That is what was meant by the original statement, and it's what the second half of the sentence more properly contrasts with.
May I suggest the following:
```suggestion
Playbooks are created inside resource groups, and are visible and applicable at the subscription level, but the **Playbooks** tab (in the new **Automation** blade) displays all the playbooks available across any selected subscriptions.
```

Reply:
@yelevin this was brought up to me by a client as it contracted the delegation model we were building.  Implying that subscription level permissions were needed to create playbooks.  I don't quite understand what is meant by 'applicable at the subscription'.

Perhaps we could go with:
Playbooks are created inside resource groups, and are visible at the subscription level, but the **Playbooks** tab (in the new **Automation** blade) displays all the playbooks available across any selected subscriptions.

Reply:
@briandelmsft This is being taken care of in a pull request that is making broader changes in the documentation. This PR can be closed.

Reply:
@briandelmsft This change is being implemented in a PR making broader changes to the documentation. This PR can now be closed.